### ISSUE TITLE
[wg/webtransport] [WebTransport] Group Charter

### ISSUE THREAD

**Issue by ylafon** [w3c/strategy#537]
New charter proposal, reviewers please take note.

# Charter Review

[Draft Charter](https://w3c.github.io/charter-drafts/2026/webtransport-wg-charter.html)

Note that is it still using an old version of the template, changes were made from the last charter, update to the new template is in progress.

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/charter-drafts/2026/webtransport-wg-charter.html)

Expected end of charter refinement phase: [2026-06-03](https://lists.w3.org/Archives/Member/w3c-ac-members/2026AprJun/0031.html)

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2024%2F09%2Fwebtransport-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2026%2Fwebtransport-wg-charter.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/webtransport)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [ ] New WG 
 - [ ] New IG
 
* Existing 
 - [X] Existing WG recharter 
 - [ ] Existing IG recharter
 
Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Communities suggested for outreach

 * HTML

## Known or potential areas of concern

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
[Use "webtransport" label](https://github.com/w3c/charter-drafts/issues)

## Anything else we should think about as we review? 

Note: proposed chairs should be copied @... on this issue.
@wilaw @jan-ivar

Note: The Technical Strategy Team Lead or Project & Process Team Lead will assign the issue to a [Charter Facilitator](https://www.w3.org/guide/process/charter.html#cr-acceptance) for new charters. For rechartering, the team contact is the Charter Facilitator by default, please assign the issue to them directly.

**Comment by dontcallmedom** [comment]
a couple of comments:
* the Web & Networks IG has closed so should be removed from the coordination list
* the expected completion date for WebTransport of Q2 2026 is both unlikely (given that Q2 ends in 3 weeks and the spec is still a WD) and odd since the group is asking to renew its charter for 2 years; likewise, the timeline section would need an update - in general, since the group has been operating for 5+ years, it might be good to document more specific plans with regard to CR at least

**Comment by himorin** [comment]
- still have Proposed Recommendation in Success criteria
- mentions to TAG Web platform design principles, but not Ethical Web Principles nor Privacy Principles (with above point, it seems replacement with current charter template may be better?)
- in Coordination -> external organizations, WebSocket link need to be replaced with new one?: https://websockets.spec.whatwg.org/
- any intention to stick with Patent Policy 2020? (but not 2025)

**Comment by ruoxiran** [comment]
no comments or request from APA.

**Comment by jyasskin** [comment]
No comments from the TAG breakout that looked at this.

**Comment by plehegar** [comment]
[Charter refinement](https://lists.w3.org/Archives/Member/w3c-ac-members/2026AprJun/0031.html)

**Comment by himorin** [comment]
No comment or request from i18n

**Comment by ylafon** [comment]
@dontcallmedom 
> a couple of comments:
> 
> * the Web & Networks IG has closed so should be removed from the coordination list
> * the expected completion date for WebTransport of Q2 2026 is both unlikely (given that Q2 ends in 3 weeks and the spec is still a WD) and odd since the group is asking to renew its charter for 2 years; likewise, the timeline section would need an update - in general, since the group has been operating for 5+ years, it might be good to document more specific plans with regard to CR at least

Also
@himorin 
> * still have Proposed Recommendation in Success criteria
> * mentions to TAG Web platform design principles, but not Ethical Web Principles nor Privacy Principles (with above point, it seems replacement with current charter template may be better?)
> * in Coordination -> external organizations, WebSocket link need to be replaced with new one?: https://websockets.spec.whatwg.org/
> * any intention to stick with Patent Policy 2020? (but not 2025)

Fixed in latest version per https://github.com/w3c/charter-drafts/commit/34f765b6bab04042d22347f870815d90704396a0
Aligned to latest Charter Template
Removed Web & Network IG
Fixed Timeline and Normative Specifiation ot match latest evolution.
Replaced link to Websocket specificatio

**Comment by plehegar** [comment]
[Proposed W3C Charter: WebTransport Working Group (until 2026-08-21)](https://lists.w3.org/Archives/Public/public-new-work/2026Jul/0004.html)

**Comment by simoneonofri** [comment]
_This is a late, non-blocking Security comment, since the charter is already in AC review._

The new [peer-to-peer scope](https://w3c.github.io/charter-drafts/2026/webtransport-wg-charter.html#scope) appears to cover the local-network and NAT-traversal work discussed in [w3c/webtransport#590](https://github.com/w3c/webtransport/issues/590). This is not the same proposal as the Local Peer-to-Peer API reviewed by TAG, but that [discussion](https://github.com/w3ctag/design-reviews/issues/932#issuecomment-2069024041) raised relevant questions about discovery, network information, fingerprinting, and abuse of the resulting communication channel.

For the future work, could you confirm that the P2P incubation will include at least a high-level review?

**Comment by ylafon** [comment]
> _This is a late, non-blocking Security comment, since the charter is already in AC review._
> 
> The new [peer-to-peer scope](https://w3c.github.io/charter-drafts/2026/webtransport-wg-charter.html#scope) appears to cover the local-network and NAT-traversal work discussed in [w3c/webtransport#590](https://github.com/w3c/webtransport/issues/590). This is not the same proposal as the Local Peer-to-Peer API reviewed by TAG, but that [discussion](https://github.com/w3ctag/design-reviews/issues/932#issuecomment-2069024041) raised relevant questions about discovery, network information, fingerprinting, and abuse of the resulting communication channel.
> 
> For the future work, could you confirm that the P2P incubation will include at least a high-level review?

As any new capability, this will get early review. Note that there should be a breakout on that at TPAC.

**Comment by ylafon** [comment]
Announced https://lists.w3.org/Archives/Member/w3c-ac-members/2026JulSep/0055.html
